libpod/container_internal.go

```diff
@@ -382,7 +385,7 @@ func (c *Container) removeConmonFiles() error {
 
 func (c *Container) export(path string) error {
 	mountPoint := c.state.Mountpoint
-	if !c.state.Mounted {
+	if !c.state.Mounted && c.config.Rootfs == "" {
```

Comment:
We should probably make Mounted always true if config.Rootfs is set. It's used as a check that the container's rootfs is accessible, and with an exploded rootfs that's always the case.

Reply:
thanks, this check is already superfluous and I am going to drop it as `c.State.Mounted` should  be set at this point.